### ISSUE TITLE
Please move credit / contact information to help menu, and template

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#227]
We currently display specific contact information for Kalani in the main interface and in nc-multiplex (see attached for net.create). 

Can we a) remove this from the main interface and put it into the top of the help menu?  And b) make this part of the templating system so that each local install can have a local help contact (or none at all). No rush on this, just thinking of it.

<img width="1721" alt="Screenshot 2024-05-31 at 10 35 58 AM" src="https://github.com/netcreateorg/netcreate-itest/assets/5529898/8619fed4-013e-47fd-9dc4-524ac71c39d6">

**Comment by benloh** [comment]
I'm guessing though that you still want some kind of IU/Kalani credit in addition to the contact info?

**Comment by jdanish** [comment]
Yeah - that is currently in help where it reads (which we should keep and update):

About Net.Create
Net.Create is funded through the [EAGER program](https://www.nsf.gov/pubs/policydocs/pappguide/nsf09_1/gpg_2.jsp#IID2) at [NSF](https://www.nsf.gov/) under award #[1848655](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1848655). [Kalani Craig](http://www.kalanicraig.com/) is the PI, with Co-PIs [Joshua Danish](http://www.joshuadanish.com/) and [Cindy Hmelo-Silver](https://education.indiana.edu/about/directory/profiles/hmelo-silver-cindy.html). Software development provided by [Inquirium](http://inquirium.net/). For more information, see [Net.Create.org](http://netcreate.org/)

**Comment by benloh** [comment]
Duplicate. Implemented with #360.
